src/main.ts

```diff
@@ -23,16 +25,28 @@ async function bootstrap() {
   const appConfig = configService.get<AppConfig>('appConfig');
   const mediaConfig = configService.get<MediaConfig>('mediaConfig');
 
-  const swaggerOptions = new DocumentBuilder()
-    .setTitle('HedgeDoc')
+  const publicApiOptions = new DocumentBuilder()
+    .setTitle('HedgeDoc Public API')
     .setVersion('2.0-dev')
```

Comment:
Shouldn't this be extracted from the package.json or something (less chance of a missed change?)

Reply:
Yeah, but we currently don't have a global version or similar.
So I'd just keep it that way for now

Reply:
Maybe add an Todo then?